test/cat.jl

```diff
@@ -94,4 +94,36 @@ for (f, d) in zip((vcat, hcat), (1, 2))
             @test dimnames(f(a, nda)) == dimnames(cat(a, nda; dims=d))
         end
     end
+    @testset "more than 2 dimensions" begin
```

Comment:
Could you please put these two `@testset`s outside the loop over `hcat` and `vcat`?

Reply:
Oh sorry, I messed that up, will fix.